pages/home.html

```diff
@@ -0,0 +1,442 @@
+            tempor.
+          </p>
+        </div>
+        <div class="feature__card">
+          <img
+            src="/assets/vectors/AI-icon.svg"
+            alt="security mark"
+            class="feature__card-image"
+          />
+          <h3 class="feature__card-title">
+            Customizable AI<br />
+            features
+          </h3>
+          <p class="feature__card-text">
+            Lorem ipsum dolor sit amet, consetetur<br />
+            sadipscing elitr, sed diam nonumy eirmod<br />
+            tempor.
+          </p>
+        </div>
+      </div>
+    </section>
+
+    <section class="customer">
+      <h1 class="customer__title">
+        The stunning results our<br />
+        customers have experienced
+      </h1>
+      <div class="customer__container">
+        <div class="customer__card">
+          <p class="customer__card-text">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
+            erat, sed diam voluptua.
+          </p>
+          <div class="customer__card-image">
+            <img src="/assets/images/person1.png" alt="photo-1" />
+            <p class="customer__card-name">Ron Wood</p>
+            <p class="customer__card-title">CEO</p>
+          </div>
+        </div>
+        <div class="customer__card">
+          <p class="customer__card-text">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
+            erat, sed diam voluptua.
+          </p>
+          <div class="customer__card-image">
+            <img src="/assets/images/person2.png" alt="photo-1" />
+            <p class="customer__card-name">Mark Mason</p>
+            <p class="customer__card-title">Marketing Manager</p>
+          </div>
+        </div>
+        <div class="customer__card">
+          <p class="customer__card-text">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
+            erat, sed diam voluptua.
+          </p>
+          <div class="customer__card-image">
+            <img src="/assets/images/person3.png" alt="photo-1" />
+            <p class="customer__card-name">Sam Preston</p>
+            <p class="customer__card-title">CTO</p>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- footer container -->
+    <section class="footer">
+      <div class="footer__content">
+        <h1 class="footer__title">
+          Are you ready to grow your<br />
+          business with us?
+        </h1>
+        <p class="footer__text">
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy.
+        </p>
+        <button class="primary-btn footer__button">View Pricing</button>
```

Comment:
The class name is not consistent with the parent block. the `primary-btn` shouldn't be used here. 

you should use a modifier and classes  would look something like this `footer__button footer__button--primary`